subworkflows/ebi-metagenomics/reads_bwamem2_decontamination/main.nf

```diff
@@ -13,7 +14,7 @@ workflow READS_BWAMEM2_DECONTAMINATION {
 
     ch_versions = Channel.empty()
 
-    BWAMEM2_MEM(ch_reads, ch_reference)
+    BWAMEM2_MEM(ch_reads, ch_reference) // TODO needs 4 inputs
```

Comment:
The other two inputs are:
```
    tuple val(meta3), path(fasta)  // This is the reference genome in fasta format 
    val   sort_bam                 // This is a boolean: use samtools sort (true) or samtools view (false)
```
Fasta is required for cram output, so as far as we don't use cram as extension for the output format arg, we could pass an empty tuple.